helm/values.yaml

```diff
@@ -1,7 +1,7 @@
 # configure connaisseur deployment
 deployment:
   replicasCount: 3
-  image: securesystemsengineering/connaisseur:v1.4.0
+  image: securesystemsengineering/connaisseur:v1.3.1
```

Comment:
that's a downgrade. Is that intended?

Reply:
it's changed in the latest commit :+1:

Reply:
I don't see why any intermediate commit would lower the version number...?

Reply:
I reverted one of Teetjes commit and reordered commits - the revert commit changed the version number.